Project-iOS/XLProjectName/XLProjectName/Models/RealmManager.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  RealmManager.swift
+//  XLProjectName
+//
+//  Created by XLAuthorName ( XLAuthorWebsite )
+//  Copyright © 2016 'XLOrganizationName'. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import Crashlytics
+import RealmSwift
+
+class RealmManager: AnyObject {
+
+    static let sharedInstance = RealmManager()
+
+    private(set) var defaultRealm: Realm!
+
+    private var config = Realm.Configuration()
+
+    private init() {
+        config.schemaVersion = 2
+        config.migrationBlock = { migration, oldSchemaVersion in
+            // Perform migrations when needed
+            if oldSchemaVersion == 1 {
+                // Added new fields to Employee: company, email, phone, positions, role, rawReachabilityStatus
+            }
+        }
+
+        do {
+            defaultRealm = try Realm(configuration: config)
+            DEBUGLog("Realm DB path: \(config.fileURL)")
+        } catch {
+            let nserror = error as NSError
+//            Crashlytics.CLS_LOG("Error while setting up realm: %@", [nserror])
+            Crashlytics.sharedInstance().recordError(nserror)
+        }
+    }
+
+    func eraseAll() {
+        do {
+            let realm = try createRealm()
+            try realm.write {
+                realm.deleteAll()
+            }
+        } catch {
+            let nserror = error as NSError
+//            Crashlytics.CLS_LOG("Error while setting up realm: %@", [nserror])
+            Crashlytics.sharedInstance().recordError(nserror)
+        }
+    }
+
+    func createRealm() throws -> Realm {
+        return try Realm(configuration: config)
+    }
+
+}
+
+extension RealmSwift.Results {
```

Comment:
it's not a good practice for big db results. I imagine that realm results works in a lazy way.